coa/pkg/apis/v1alpha2/providers/states/memorystate/memorystate.go

```diff
@@ -436,6 +436,10 @@ func (s *MemoryStateProvider) Get(ctx context.Context, request states.GetRequest
 	return states.StateEntry{}, err
 }
 
+func (s *MemoryStateProvider) GetLatest(ctx context.Context, request states.GetRequest) (states.StateEntry, error) {
+	return states.StateEntry{}, v1alpha2.NewCOAError(nil, "Memory state store get latest is not implemented", v1alpha2.NotImplemented)
```

Comment:
I think we can have this implemented for our unit tests.

Reply:
UT and tests will be improved but no in this PR.

Reply:
> UT and tests will be improved but no in this PR.

I don't agree we can merge this PR without tests.